### DISCUSSION TITLE
Nothing is working!!! Tokens or Passwords PLZ HELP

### DISCUSSION THREAD

**Discussion by Whiteka16** [community/community#114643] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am far beyond frustrated. I am working on the linux terminal on a chromebook. I have been trying to push something into my repository via the terminal using $git push --set-upstream origin. Once I enter the command I am prompted to enter my username and password. No matter how many times i enter it wont work. I have cleared my saved passwords AND created a personal access token BC I also cannot figure out how to disable to 2fa. 

Please I could use some help!

TIA!!

**Comment by benogoulart** [comment]
Hi,
GitHub no longer accepts account passwords for Git operations over HTTPS. If you're prompted for a password, you need to use a **Personal Access Token (PAT)** instead of your GitHub account password.

A few things to check:
* Make sure you're entering your **GitHub username** when prompted for the username.
* When prompted for the password, paste your **Personal Access Token**, not your GitHub password.
* Verify that your PAT hasn't expired and has the required permissions (for example, `repo` for private repositories).
* You can also check your remote URL with:
  ```bash
  git remote -v
  ```
  to confirm you're pushing to the correct repository.
If you're still having trouble, could you share:
* The exact error message you receive after entering your credentials?
* The output of `git remote -v` (feel free to hide your username if you prefer).
That will make it much easier to identify what's going wrong.
